dragonchain/broadcast_processor/broadcast_functions.py

```diff
@@ -109,6 +111,24 @@ def increment_storage_error_sync(block_id: str, current_level: int) -> None:
     return await redis.z_range_by_score_async(IN_FLIGHT_KEY, 0, int(time.time()), withscores=True, offset=0, count=1000)
 
 
+async def get_notification_verifications_for_broadcast_async() -> List[str]:
+    """Get the notifications scheduled to be checked by the broadcast processor right now
+    Args:
+    Returns:
+        List of notifications to be re-transmitted by the broadcast processor (limit of 1000)
+    """
+    return await redis.lrange_async(NOTIFICATION_KEY, 0, 1000)
+
+
+async def remove_notification_verification_for_broadcast_async(value: str) -> int:
+    """Removes the notification object
+    Args:
+    Returns:
+        Number of elements removed from the notification verification list
+    """
+    return await redis.lrem_async(NOTIFICATION_KEY, value)
```

Comment:
I think this is wrong. If your goal is to remove a single instance of `value` in the redis list, you definitely have to pass in a count of `1` somewhere on top of your value.